app/mailers/meeting_invitation_mailer.rb

```diff
@@ -7,7 +7,8 @@ class MeetingInvitationMailer < ActionMailer::Base
   def attending meeting, member, invitation
     @member = member
     @meeting = meeting
-    @host_address = AddressDecorator.new(@meeting.venue.address) 
+    @host_address = AddressDecorator.new(@meeting.venue.address)
+    @cancellation_url = 'https://codebar.io' + meeting_path(@meeting)
```

Comment:
try meetings_url(@meeting) instead of path. You shouldnt need to hardcode 'codebar.io'